dlt/common/configuration/exceptions.py

```diff
@@ -73,29 +73,34 @@ def __str__(self) -> str:
                     " missing or empty.\n"
                 )
 
-        # check if entry point is run with path. this is common problem so warn the user
         main_path = main_module_file_path()
-        if main_path and main_path.endswith(".py"):
+
+        # check if entry point is run with path or from cli command. This is common problem so warn the user
+        if main_path and (main_path.endswith(".py") or main_path.endswith("dlt")):
             from dlt.common.runtime import run_context
+            import dlt
+
+            last_ctx = dlt.current.pipeline().get_local_state_val("last_run_context")
+            last_ctx_local_dir = last_ctx.get("local_dir")
```

Comment:
mind that `last_ctx` may be None here! (if pipeline never fully ran). with changes I proposed below you'll be able to get this as a pipeline instance field. still it may be `None`